src/backend/distributed/connection/connection_management.c

```diff
@@ -625,6 +628,52 @@ StartConnectionEstablishment(ConnectionHashKey *key)
 
 
 /*
+ * CitusSSLModeString returns the current value of citus.sslmode.
+ */
+char *
+CitusSSLModeString(void)
+{
+	switch (CitusSSLMode)
+	{
+		case CITUS_SSL_MODE_DISABLE:
+		{
+			return "disable";
```

Comment:
I searched for the equivalent of this `switch` in Postgres source in case we can directly use that. But, I couldn't find any direct equivalent of this.